src/PowerShellEditorServices/Services/PowerShellContext/Handlers/ExpandAliasHandler.cs

```diff
@@ -69,7 +69,8 @@ public async Task<ExpandAliasResult> Handle(ExpandAliasParams request, Cancellat
                 .AddStatement()
                 .AddCommand("__Expand-Alias")
                 .AddArgument(request.Text);
-            var result = await _powerShellContextService.ExecuteCommandAsync<string>(psCommand).ConfigureAwait(false);
+            var result = await _powerShellContextService.ExecuteCommandAsync<string>(
+                psCommand, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
                psCommand, cancellationToken).ConfigureAwait(false);
```

Reply:
Can't, `ExecuteCommandAsync` takes other args before `cancellationToken` so it has to be named.